Dockerfile

```diff
@@ -1,6 +1,27 @@
+# Get python version
+FROM python:3.6-slim
+
+# Install mantid image
 FROM marshallmcdonnell/mantid:latest
```

Comment:
Should this just move to use conda and carry on? https://anaconda.org/mantid/mantid-framework

Reply:
From slack discussion, we will need to discuss more since @ElliotAOram was also trying conda in tandem. Docker was the fastest to solution.